### DISCUSSION TITLE
[firebase_analytics] AnalyticsWithoutAdIdSupport

### DISCUSSION THREAD

**Discussion by pinpong** [firebase/flutterfire#6030] [Feature request]
Hello,

the flutter plugin is currently not support the latest feature from Analytics 7.11.0
https://firebase.google.com/support/release-notes/ios#analytics

Checkout react native version for more information:

https://github.com/invertase/react-native-firebase/commit/da6b811e15b480ad55c1e804da40387ecfdef3ee

**Comment by softkot** [comment]
Offer the MR https://github.com/FirebaseExtended/flutterfire/pull/6046
